app/main.ipynb

```diff
@@ -151,17 +271,47 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "df[\"order_day_of_week\"] = df[\"datetime\"].dt.dayofweek\n",
+    "\n",
+    "daily_income = df.groupby(\"order_day_of_week\")[\"item_price\"].sum()\n",
+    "\n",
+    "plt.bar(daily_income.index, daily_income.values)\n",
+    "plt.xlabel(\"Order Day Of The Week\")\n",
+    "plt.ylabel(\"Total Income\")\n",
+    "plt.title(\"Restaurant Income by Order Day Of The Week\")\n",
+    "plt.xticks(range(7), [\"Mon\", \"Tue\", \"Wed\", \"Thu\", \"Fri\", \"Sat\", \"Sun\"])\n",
+    "plt.show()"
    ],
    "metadata": {
     "collapsed": false,
     "pycharm": {
      "name": "#%%\n"
+    },
+    "ExecuteTime": {
+     "end_time": "2024-05-02T10:29:53.892282Z",
+     "start_time": "2024-05-02T10:29:51.378568Z"
     }
    }
+  },
+  {
+   "cell_type": "code",
+   "outputs": [],
+   "source": [],
+   "metadata": {
+    "collapsed": false
```

Comment:
Still cannot view, I suppose it's because here is missing "execution_count": 1 after cell_type